src/main/resources/jenkins/branch/BranchSource/config.jelly

```diff
@@ -35,8 +35,10 @@
       </table>
     </td>
   </tr>
-  <f:block>
-    <f:dropdownDescriptorSelector field="strategy" title="Property strategy"
-                                  descriptors="${descriptor.propertyStrategyDescriptors(it,sourceDescriptor)}"/>
-  </f:block>
+  <j:set var="descriptors" value="${descriptor.propertyStrategyDescriptors(it,sourceDescriptor)}"/>
+  <j:if test="${!descriptors.isEmpty()}">
+    <f:block>
+      <branch:dropdownDescriptorSelector field="strategy" title="Property strategy" descriptors="${descriptors}"/>
```

Comment:
> is this supposed to have a capture attribute?

No, because the default is sensible.



> why include the unused attribute in the temporary file?

It is simply copied from Jenkins trunk sources, with only my `TODO` comment added.